target/linux/econet/dts/en751221.dtsi

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/dts-v1/;
+
+/ {
+	compatible = "econet,en751221";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	hpt_clock: clock {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-frequency = <200000000>;  /* 200 MHz */
+	};
+
+	cpus: cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "mips,mips24KEc";
```

Comment:
I believe it is mips 34Kc.

Reply:
MT7620 is a MIPS 24Kc core. MIPS 34Kc is used in Realtek RTL839x, Lantiq xRX200, etc. This string currently isn't used in the kernel, so it doesn't affect anything.

Reply:
good decision